macOS/LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Shared/UIWeb/UIMapper.swift

```diff
@@ -238,6 +238,7 @@ struct MapperToUI {
         do {
             let encoder = JSONEncoder()
             encoder.outputFormatting = .prettyPrinted
+            encoder.outputFormatting = .sortedKeys
```

Comment:
I was having a hard time following all the metadata log entries - using `sortedKeys` means that the logged JSON blob isn't constantly shifting around.

Reply:
Hurray! Did you look at any of the rest of UIMapper? It's not my favorite class 😅

Reply:
Not in too much depth yet but I do want to poke around in there 😄